### ISSUE TITLE
Some Bugs

### ISSUE THREAD

**Issue by ozilxu** [smhadisadati/TMTDyn#4]
Hi Dr. Sadati, I have tried the code from your GitHub repository. And I'm wondering if you could give me some advice to solve the below bugs.

Now I'm using v6_Latest_Version_Examples.zip and trying to run all the examples but I'm encountering some bugs.
1)For example Exp1_SRL.m I got the following bug: Error using sym/matlabFunction>computeVarnames (line 902) Dimensions of arrays being concatenated are not consistent.
2)For examples 2 and 3 I got the bugs: Check for incorrect argument data type or missing argument in call to function 'codegen'. Error in save_eom_mex (line 527)  codegen EOM_eq -args vars_mex.
3)For examples 4 and 5 I got: Error using sym/matlabFunction>computeVarnames (line 902)
Dimensions of arrays being concatenated are not consistent.

I'm using MATLAB 2021a on a Windows laptop and wonder if you have some solutions to solve those bugs.

Thanks for the help in advance! @smhadisadati

**Comment by smhadisadati** [comment]
Hi, @ozilxu,
- For each example, an "eom - xxx" folder is provided based on the mapping between the robot inputs and the modeling states in that example. Please replace the content of the "eom" folder by the content of the relevent "eom - xxx" folder before running each of the examples.
- Matlab's newer versions have hard time converting large Matlab functions time to the more efficient mex format that is implemented in the package. That is a source of issue with some of the users. Please try simplifying the model complexity by amending the folliwing part of the code and use 'assume_small_velocities' to neglect the Coriolis inertial terms
.derive_eom('assume_small_velocities')...

**Comment by ozilxu** [comment]
Hi Dr. Sadati, Thanks again for your time and effort! If I want to derive the full system for the real dynamic task, is it still possible to conduct this in the future?

**Comment by smhadisadati** [comment]
Yes, of course. You need to choose a simpler framework that results in simpler final derivations. For that, I suggest EBR (which is similar to FEM kinematics, also know as ANBF) or ROM kinematics.
